### DISCUSSION TITLE
Self hosted Deployments and PGadmin Connections

### DISCUSSION THREAD

**Discussion by rdmorris** [digitalocean/supabase-on-do#18] [Q&A] [answered]
According to the [docs here,](https://supabase.com/docs/guides/database/connecting-to-postgres) you need to access the direct connection info via the settings->database menu. However on my self hosted deployment I have no option for settings. Any ideas? Should I reconfigure the docker compose file for this connectivity?

**Comment by timothymamo** [comment, marked as answer]
@rdmorris 

You can open port 5432 by setting the variable [`enable_db_con`](https://github.com/digitalocean/supabase-on-do/blob/main/terraform/variables.tf#L168) to `true`. This will allow you to connect to the database using the password obtained by running the command `terraform output psql_pass` within the terraform directory.

I would also suggest to set a [range of IPs](https://github.com/digitalocean/supabase-on-do/blob/main/terraform/variables.tf#L147) to restrict access to the database for security reasons.

The documentation you reference is mostly for the managed service offered by Supabase rather than the self-hosted option. So you won't see a Settings option.
